src/main/docker/env.bash

```diff
@@ -8,6 +8,10 @@ export JENKINS_PORT=8082
 export MAIL_SMTP_PORT=1025
 export MAIL_WEB_PORT=8025
 export EI_BACKEND_PORT=8090
+export ELASTICSEARCH_PORT1=8087
```

Comment:
Please change elastic search ports to some other less used ports :) If I have problems with allocated ports (even after purging docker containers...) then perhaps others also have the same issue.